src/controllers/itemdetailpage.js

```diff
@@ -506,15 +508,15 @@ define(["loading", "appRouter", "layoutManager", "connectionManager", "cardBuild
                 index: 0,
                 tag: item.ImageTags.Thumb
             });
-            itemBackdropElement.classList.remove("noBackdrop");
+            page.classList.remove("noBackdrop");
             imageLoader.lazyImage(itemBackdropElement, imgUrl, false);
             hasbackdrop = true;
         } else {
-            itemBackdropElement.classList.add("noBackdrop");
             itemBackdropElement.style.backgroundImage = "";
         }
 
         if ("Person" === item.Type) {
+            page.classList.add('noBackdrop');
```

Comment:
why?

Reply:
Person's seldom have a backdrop. There's likely a better way to handle this case, though.

Reply:
add a `//FIXME` comment explaining this case then, please